platform/viewer/src/loadDynamicConfig.js

```diff
@@ -0,0 +1,23 @@
+export default async config => {
+  const useDynamicConfig = config.dangerouslyUseDynamicConfig;
+
+  // Check if dangerouslyUseDynamicConfig enabled
+  if (useDynamicConfig.enabled) {
```

Comment:
This has to be useDynamicConfig?.enabled so that if the value isn't configured, it won't throw.